.github/workflows/terraform-integration-tests.yml

```diff
@@ -57,7 +57,7 @@ jobs:
           if [[ $hz_version == *"SNAPSHOT"* ]]; then
             wo_SNAPSHOT=$( echo $hz_version | grep -Po "[.a-zA-Z0-9]+(?=-SNAPSHOT)" )
             snapshot_link=$(curl -s "https://oss.sonatype.org/content/repositories/snapshots/com/hazelcast/hazelcast/$hz_version/" 2>&1 \
-                 | grep -Po  "hazelcast-$wo_SNAPSHOT-\d{8}.\d{6}-\d+\.jar" | head -1)
+                 | grep -Po  "hazelcast-$wo_SNAPSHOT-\d{8}.\d{6}-\d+\.jar" | tail -1)
```

Comment:
LGTM 👍 Just an advice, `mvn` has already installed your machine so you can use below approach to download latest SNAPSHOT or latest jar:
```
export HZ_JAR_REPO_URL="https://oss.sonatype.org/content/repositories/snapshots"
export HZ_JAR_REPO_URL="https://repo1.maven.org/maven2"

mvn dependency:get -DgroupId=com.hazelcast -DartifactId=hazelcast -Dversion=${HZ_VERSION} \
        -Dpackaging=jar -DremoteRepositories=${HZ_JAR_REPO_URL} -Ddest=./hazelcast-${HZ_VERSION}.jar 
```

Reply:
I think this would be better to use. But it says "-Dtest" is deprecated. Will there be a new option?

Reply:
I could not find replacement for it at its doc --> https://maven.apache.org/plugins/maven-dependency-plugin/get-mojo.html

If you do not specify `-Ddest` param, the jar downloaded into `~/.m2/repository/com/hazelcast/hazelcast/4.1-SNAPSHOT/hazelcast-4.1-SNAPSHOT.jar`.